src/styles/theme.js

```diff
@@ -16,14 +16,13 @@ const theme = {
         primaryLight: "#2e3438",
         secondaryLight: "#f9db9c",
         secondary: "#ffad00",
-        // secondaryDark: "#d99300",
+        // secondaryDarkVar: "#d99300",
```

Comment:
if we don't need this, then delete it :)

Reply:
i am gonna use it. i saved it there :)